include/ddc/kernels/splines/view.hpp

```diff
@@ -75,17 +70,17 @@ std::ostream& stream_impl(
 // template <class>
 // struct IsContiguous;
 // template <>
-// struct IsContiguous<std::experimental::layout_stride>
+// struct IsContiguous<Kokkos::layout_stride>
 // {
 //     static constexpr bool val = false;
 // };
 // template <>
-// struct IsContiguous<std::experimental::layout_right>
+// struct IsContiguous<Kokkos::layout_right>
 // {
 //     static constexpr bool val = true;
 // };
 // template <class ElementType, class Extents, class LayoutPolicy, class AccessorPolicy>
-// struct IsContiguous<std::experimental::mdspan<ElementType, Extents, LayoutPolicy, AccessorPolicy>>
+// struct IsContiguous<Kokkos::mdspan<ElementType, Extents, LayoutPolicy, AccessorPolicy>>
 // {
 //     static constexpr bool val = IsContiguous<LayoutPolicy>::val;
 // };
```

Comment:
Why do you keep these lines?

Reply:
Actually I did not even remember this was there. Will be removed.

Reply:
see #657